setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup
+
+setup(
+    author="CS50",
+    author_email="sysadmins@cs50.harvard.edu",
+    classifiers=[
+        "Intended Audience :: Developers",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Topic :: Software Development :: Libraries :: Python Modules"
+    ],
+    description="CS50 library for Python",
+    install_requires=["SQLAlchemy"],
+    keywords="cs50",
+    name="cs50",
+    packages=["cs50"],
+    url="https://github.com/cs50/python-cs50",
+    version="1.3.0"
```

Comment:
@dmalan ended up reading the version specified in `setup.py` [this](https://github.com/cs50/submit50/blob/912beca53e1fce0e87fb2697fd5932a2156979c6/submit50#L30) way, without need to factor it out. Generally, I don't think we need to factor it out so long as we don't build multiple distributions (e.g., deb, rpm, sdist, etc).

Factoring it out (e.g., to a file) and parsing that from `setup.py` (or any number of other scripts/config files) is fairly easy, but just feels unnecessary to me, tbh, specially that we decided to distribute Python packages via PyPI only? Thoughts?